setup.py

```diff
@@ -47,12 +47,18 @@
         "ipyparallel": ["ipyparallel >= 6.2.4", "dill >= 0.3.3"],
         "cuda": ["cupy >= 10.0.0"],
         "acom": ["pymatgen >= 2022", "mp-api == 0.24.1"],
-        "aiml": ["tensorflow == 2.4.1", "tensorflow-addons <= 0.14.0", "crystal4D"],
+        "aiml": [
+            "tensorflow <= 2.10.0",
+            "tensorflow-addons <= 0.16.1",
+            "crystal4D",
+            "typeguard == 2.7",
+        ],
         "aiml-cuda": [
-            "tensorflow == 2.4.1",
-            "tensorflow-addons <= 0.14.0",
+            "tensorflow <= 2.10.0",
+            "tensorflow-addons <= 0.16.1",
             "crystal4D",
             "cupy >= 10.0.0",
+            "typeguard == 2.7",
```

Comment:
What is `typeguard`? It doesn't seem to be used anywhere...

Reply:
It's a hack to get tensorflow-addons to install correctly